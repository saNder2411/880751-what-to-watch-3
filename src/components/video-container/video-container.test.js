import React from 'react';
import renderer from 'react-test-renderer';
import VideoContainer from './video-container.jsx';

const videoProps = {
  src: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  poster: `img/bohemian-rhapsody.jpg`,
  isPlaying: true,
  isMuted: false,
  isDelay: false,
  width: 280,
  height: 175,
};

it(`VideoPlayer is rendered correctly`, () => {

  const markup = renderer.create(
      <VideoContainer {...videoProps}>
        <video/>
      </VideoContainer>,
      {
        createNodeMock: () => {
          return {};
        }
      }
  ).toJSON();

  expect(markup).toMatchSnapshot();
});
