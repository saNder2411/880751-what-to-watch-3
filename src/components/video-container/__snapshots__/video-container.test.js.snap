// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer is rendered correctly in PreviewCard 1`] = `<video />`;

exports[`VideoPlayer is rendered correctly in VideoPlayerScreen 1`] = `<video />`;
